src/index.scss

```diff
@@ -0,0 +1,139 @@
+/* =================================
+  WEB FONTS
+==================================== */
+
+@import url(https://fonts.googleapis.com/css?family=Roboto:400,500,700);
+
+/* =================================
+  ELEMENT STYLES
+==================================== */
+
+* {
+  box-sizing: border-box;
+}
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-color: #e8e7e7;
+}
+h1,
+h2,
+ul {
+  margin: 0;
+  padding: 0;
+}
+h2 {
+  color: #404040;
+  font-size: 1.6em;
+  margin-bottom: 1.25em;
+}
+ul {
+  list-style-type: none;
+}
+a {
+  text-decoration: none;
+  color: #3b80d1;
+}
+hr {
+  border: 0;
+  height: 0;
+  margin: 2.3em 0 2em 0;
+  border-top: 1px solid rgba(0, 0, 0, .1);
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+    monospace;
+}
+
+.page {
+  padding: 100px 0px;
+
+  .container {
+    width: 100%;
+    max-width: 960px;
+    margin: 0 auto 2em;
+    background: #fff;
+    border-radius: .5em;
+    box-shadow: 0 3px rgba(0,0,0, .1);
+    overflow: hidden;
+  }
+
+  header:not(.page-score-board__header) {
+    background: #edeff0;
+  }
+
+  .main-nav {
+    width: 100%;
+
+    a {
+      font-size: .95em;
+      color: #222;
+      text-align: center;
+      text-transform: uppercase;
+      display: block;
+      padding: 1em;
+      transition: .2s;
+
+      &:hover,
+      &.active {
+        color: #fff;
+        background: #3b80d1;
+      }
+    }
+  }
+
+  .main-content {
+    padding: 1em 2em;
+  }
+
+  .content {
+    &-wrapper {
+      padding: 24px;
+    }
+
+    &-header {
+      margin-bottom: 24px;
+
+      &__title {
+        font-size: 24px;
+      }
+    }
+  }
+
+  @media (min-width: 0) and (max-width: 768px) {
```

Comment:
Same here. No need for the `min-width: 0`.

Reply:
updated, removed min-width: 0